common/views/components/PageLayout/PageLayout.tsx

```diff
@@ -2,7 +2,6 @@ import React, { useContext, FunctionComponent, PropsWithChildren } from 'react';
 import { Url } from '../../../model/link-props';
 import { JsonLdObj } from '../JsonLd/JsonLd';
 import Head from 'next/head';
-import Script from 'next/script';
```

Comment:
I think this will break when you merge into main; although it was unused at one point (when I replaced it with `<script>` inside the `<Head>`), the PR I did this morning restores its use.